vNext/extensions/applicationinsights-analytics-js/src/JavaScriptSDK/ApplicationInsights.ts

```diff
@@ -92,6 +92,19 @@ export class ApplicationInsights implements IAppInsights, ITelemetryPlugin, IApp
         return config;
     }
 
+    private static _createDomEvent(eventName: string): Event {
```

Comment:
Putting this in a Util class instead would be better

Reply:
Moved to common's `Util`